ls1intum/Artemis
# [Bugfix] Fix assessment after complaint for programming exercises

### Checklist

### Motivation and Context



When updating the manual assessment of a programming exercise after a complaint, the test case and sca feedback does not get displayed correctly. Failed test cases are marked as 'passed' and sca feedback is displayed as tutor feedback:

Fixes #2291 

### Description


For programming exercises we now skip updating the automatic feedback when updating a result after a complaint.

### Steps for Testing


1. Log in to Artemis
2. Create a programming exercise with manual assessment
3. Participate in the programming exercise, assess this as an instructor, complain as the student.
4. Re-assess as the instructor addressing the complaint
5. Check that the test-case and sca feedback is displayed correctly, as well as all other feedback.

### Test Coverage


Nothing changed.

### Screenshots



For screenshots see #2177
